comp110-worksheet-6/OxoBoard.cs

```diff
@@ -1,47 +1,122 @@
-﻿using System;
+using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
 
 namespace comp110_worksheet_6
 {
-	public enum Mark { None, O, X };
+public enum Mark { None, O, X };
+ 
 
-	public class OxoBoard
-	{
-		// Constructor. Perform any necessary data initialisation here.
-		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+    public class OxoBoard
+{
+        public int width = 3;
+        public int height = 3;
+        public int inARow = 3;
+        int[,] myGrid = new int[width, height];
```

Comment:
`width` and `height` can't be used here as they're variables, not constants. You could instead just declare
```
int[,] myGrid;
```
here, and then in the constructor do
```
myGrid = new int[width, height];
```